### PR TITLE
Add refs for firewall relationship

### PR DESCRIPTION
**Pull request by alero-awani**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: server/meshmodel/kubernetes/v1.32.0-alpha.1/v1.0.0/relationships/edge-binding-firewall-kllzw.json [hunk 1]
```diff
@@ -63,7 +67,10 @@
                   "id": null,
                   "kind": "NetworkPolicy",
                   "mutatorRef": [
-                    []
+                    ["configuration", "spec", "podSelector", "matchLabels"],
+                    ["configuration", "spec", "policyTypes","_"],
+                    ["configuration", "spec", "ingress", "_", "from", "_", "podSelector", "matchLabels"],
```
Comment (Jougan-0) [inline server/meshmodel/kubernetes/v1.32.0-alpha.1/v1.0.0/relationships/edge-binding-firewall-kllzw.json:72]: This won't work there needs to be a single `_` in a set.

**Comment by leecalcote** [comment]
merge conflict...

File: server/meshmodel/kubernetes/v1.32.0-alpha.1/v1.0.0/relationships/edge-binding-firewall-kllzw.json (hunk 1 above)
Comment (Alero-Awani) [reply server/meshmodel/kubernetes/v1.32.0-alpha.1/v1.0.0/relationships/edge-binding-firewall-kllzw.json:72]: ohh okay let me review this

**Comment by leecalcote** [comment]
@Alero-Awani, are we pushing forward with this PR?

**Comment by leecalcote** [comment]
Not hearing back on this today, I'll consider it abandoned.

**Comment by Alero-Awani** [comment]
@leecalcote I made this PR before learning how to test with the OPA CLI and actually run the evaluation locally. I will have to test it first before moving forward.

**Comment by leecalcote** [comment]
Ok, understood. Thank you for this update, @Alero-Awani.

**Pull request closed without merging**